### PR TITLE
add extension support

### PR DESCRIPTION
**Pull request by ZeroChaos-**
add extensions like bash bunny

### CODE REVIEW

**Comment by ZeroChaos-** [comment]
the file moves are successfully tested, need to actually test adding an extension before merge, just to be sure

**Comment by ZeroChaos-** [comment]
something is wrong, logging enough to find out what is wrong has been a problem, so don't merge this one yet.

**Comment by ZeroChaos-** [comment]
Okay, this works now.  I realize I rearchitected a little, but it feels a lot cleaner to do it this way.  The other alternative was to simply not background the payloads but there are reasons to want to do that...

File: root/payload/extensions/README
Comment (foxtrot) [inline root/payload/extensions/README]: The*

**Review by foxtrot — APPROVED** [review]

**Comment by foxtrot** [comment]
Thank you!

**Pull request merged**
